### PR TITLE
Ola

### PR DESCRIPTION
**Pull request by leealex88**
Please see the title of the commits below. 

Thank you for contributing! To help the review process, please provide the following:

### Proposal
Provide a short description of the change you are proposing and the reasons for it.

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: client/src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { Link } from "react-router-dom";
+
+import "./navbar.css";
+
+const Navbar = () => {
+	return <div className="navbar-container">
+		<Link
+			className="navbar-route-link"
+			to={{ pathname: "/students" }}
+		><h3>Feedbak Tracker</h3></Link>
```
Comment (mnashawati) [inline client/src/components/Navbar.js:11]: Please amend the heading to "Feedback Tracker"

File: client/src/components/StudentProfile.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
+/* eslint-disable jsx-a11y/alt-text */
```
Comment (mnashawati) [inline client/src/components/StudentProfile.js:1]: Could you please try to remove these "eslint" comments before merging?

**Review by mnashawati — APPROVED** [review]
Looks great, good job Ola!

**Review by tayfuntayfun — APPROVED** [review]
Thanks Ola.

**Pull request merged**